### ISSUE TITLE
Removed

### ISSUE THREAD

**Issue by jslegers** [LeaVerou/multirange#27]
Removed

**Comment by jslegers** [comment]
I submitted a pull request to fix this issue @ https://github.com/LeaVerou/multirange/pull/30

**Comment by nicknack23** [comment]
I came here because I was confused that the demo was completely nonfunctional, immediately found this issue and saw this:

>  jslegers added a commit to jslegers/multirange that referenced this issue 42 seconds ago

Great timing for the fix @jslegers, thanks! :)

**Comment by LeaVerou** [comment]
Except I wish @jslegers hadn't submitted a mega-PR that fixes ALL THE THINGS *and* changes indentation.
Small PRs people! I would've merged this fix in a few seconds if it didn't come with all the other baggage.

**Comment by jslegers** [comment]
I tried to keep the PR as small as possible, but I'm working on a tight deadline here and I needed to get these issues fixed ASAP.

Either way, your indentation was messed up to begin with with before my changes. More specifically, your code for `self.multirange = function(input) { ... }` didn't indent at all & neither did the if-else statement at the end. IMO that's a far greater issue than whether to use tabs or spaces :-p

Anyway, why not just accept my changes and re-do the indentation the way you like it? At least with my changes your demo works as expected!
